tests/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+include(FetchContent)
+set(FETCHCONTENT_QUIET OFF CACHE BOOL "" FORCE)
+
+FetchContent_Declare(
+  Boost
+  GIT_REPOSITORY https://github.com/nxxm/boost.git
+  # boost-1.89.0
```

Comment:
please note that this is a PATCHED boost 1.89 on top of `ef7fea34711a189472893b88205b1dd3c275677b` which has the official release tag